src/app/pages/psbt-request/components/psbt-decoded-request/psbt-decoded-request-node/psbt-unsigned-input-list/psbt-unsigned-input-list.tsx

```diff
@@ -12,32 +13,33 @@ import { PsbtUnsignedInputItem } from './components/psbt-unsigned-input-item';
 interface PsbtUnsignedInputListProps {
   addressNativeSegwit: string;
   addressTaproot: string;
-  inputs: btc.TransactionInputRequired[];
   showPlaceholder: boolean;
+  utxos: (BitcoinTransactionVectorOutput | undefined)[];
+  utxosWithInscriptions: ((TaprootUtxo & OrdApiInscriptionTxOutput) | undefined)[];
```

Comment:
Can you explain the data structure of this? Do the UTXOs with inscriptions also show up in UTXOs? Are we relying on array index to present the?

If we have 3 inputs, with 1 inscription on index 1, does this yield an input of:
```
utxo = [utxo0, undefined, utxo2]
utxosWithInscriptions = [undefined, utxo1, undefined]?
```

Reply:
Yep, correct.